packages/edit-navigation/src/components/notices/style.scss

```diff
@@ -2,6 +2,7 @@
 	position: fixed;
 	bottom: 20px;
 	margin-left: 20px;
+	pointer-events: none;
```

Comment:
In previous reviews I didn't catch that this is only added in the navigation editor.

I think it should be moved to the `components-snackbar-list` style here:
https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/snackbar/style.scss#L75

That would make sure the fix works consistently across all editors.